app/views/landing_page/_hero.erb

```diff
@@ -19,11 +19,18 @@
   }
 <% end %>
 
+<%
+# Add non-breaking space to prevent widows
+subtitle_splitted = s["subtitle"]["value"].split(" ")
```

Comment:
Yep, it seems so. I tested with this text:

```
SuperlongtextwithnobreaksSuperlongtextwithnobreaks-Superlongtextwithnobreaks
```

and the browser indeed added a linebreak where the hyphen is. But can we do anything about it? We can't replace hyphens with `&nbsp;` because then we would have space instead of hyphen. On the other hand, we can't add `&nbsp;` where the hyphen is, because then we would have both, hyphen and space.